usaspending_api/common/elasticsearch/client.py

```diff
@@ -22,8 +22,9 @@ def create_es_client():
     try:
         # If the connection string is using SSL with localhost, disable verifying
         # the certificates to allow testing in a development environment
-        if settings.ES_HOSTNAME.startswith("https://localhost"):
-            logger.warn("SSL cert verification is disabled. Safe only for local development")
+        if settings.ES_HOSTNAME.startswith("https://localhost") or settings.ES_HOSTNAME.startswith(
+                "https://host.docker.internal"):
```

Comment:
@kbard wow, I learn something new every day.

Perhaps there is a better way to disable SSL cert verification without looking for specific hostname patterns. It would be outside the scope of this ticket to solve.

Reply:
Done